### ISSUE TITLE
Add link to Permissions Documentation on Roles page

### ISSUE THREAD

**Issue by jrtricafort** [ushahidi/platform#3078]
Current Behavior: 
As a user, when I edit Settings/Roles, I can see a list of Permissions checkboxes. But I don't know what the Permissions actually mean, aside from the title of the permission.

![image](https://user-images.githubusercontent.com/37776843/41365967-d24ad1c8-6f08-11e8-939b-b9bf1136851d.png)

Solution:
- [ ] Copy that sits above the checkbox lists, under the title 'Permissions' should be italic text and read:
`These options control what this type of role can make changes to. Visit our support guides to find out more.`   with 'Visit our support guides to find out more.' being a link to https://www.ushahidi.com/support/custom-roles#add-a-role

------- 

Context: This issue was influenced by https://github.com/ushahidi/platform/issues/2491 but really is something basic we should do.

### Test script
- [x] Go to the roles page as an admin
- [x] Go to create role page
- [x] In the create role page, you should see a list of permissions
- [x] Under "permissions" title, but above the checklists, you should see:
These options control what this type of role can make changes to. [Visit our support guides to find out.](https://www.ushahidi.com/support/custom-roles#add-a-role)
    - [x] The text 'Visit our support guides to find out more.'  should be a link to https://www.ushahidi.com/support/custom-roles#add-a-role

**Comment by jrtricafort** [comment]
@Erioldoesdesign Is this something you could provide some quick copy on? We just need one sentence (or maybe a phrase/not even a full sentence) providing a link to our support documentation from this page.

@rowasc This is what we discussed in relation to Manage Collections.

**Comment by Erioldoesdesign** [comment]
Well first things first, is there a reason we refer to 'posts' in the management and support docs when in the deployments they are titled as 'reports'? We have a few places where consistency is iffy with our terminology

Copy that sits above the checkbox lists, under the title 'Permissions' should be italic text and something like: 
'These options control what this type of role can make changes to. [Visit our support guides to find out more.](https://www.ushahidi.com/support/custom-roles)

That might not be the right link you want to deep link into but it looks to be the most relevant. @aoduor @StaicyG tagging Angie/Staicy to double check this language makes sense to someone who is in support most often.

**Comment by aoduor** [comment]
Hey @Erioldoesdesign thanks for flagging this. When we switched to v3, we changed terminology to posts. There should be no reference of reports anywhere on platform/support docs.

**Comment by Erioldoesdesign** [comment]
![screen shot 2018-06-19 at 16 36 41](https://user-images.githubusercontent.com/11681324/41607959-2cc50882-73df-11e8-9755-4946f0f3a12f.png)

In my live test deployment there's 'reports' at the top which should be changed at some point it seems :)

**Comment by Erioldoesdesign** [comment]
Just finishing this with a final call on copy then:

Copy that sits above the checkbox lists, under the title 'Permissions' should be italic text and read: 

'These options control what this type of role can make changes to. Visit our support guides to find out more.'

with 'Visit our support guides to find out more.' being a link to: https://www.ushahidi.com/support/custom-roles#add-a-role

**Comment by rowasc** [comment]
Nice, I think this would be pretty helpful @Erioldoesdesign  . 
@jrtricafort should I create a ticket for the UI to be changed or is there one already?

**Comment by jrtricafort** [comment]
Thanks @Erioldoesdesign ! Great catch on reports vs. posts. I'll make another ticket for us to remove any UX reference to "reports" and make sure it is replaced consistently with "posts". Small change but super important since it's the "primary object" in the platform.

@rowasc I figured it could jsut be included in this issue, but if we need another issue then feel free to create one.

**Comment by jrtricafort** [comment]
@Erioldoesdesign @rowasc Added here: https://github.com/ushahidi/platform/issues/3105

**Comment by rowasc** [comment]
@crcommons can you review?

**Comment by rowasc** [comment]
I think this is a 1

**Comment by willdoran** [comment]
@rowasc I agree

**Comment by crcommons** [comment]
@rowasc Agreed.

**Comment by Eve-wanderi** [comment]
@rowasc I get this error when i try to upgrade my deployment http://targetedsurveytesting.v3-qa.ush.zone/views/map

![image](https://user-images.githubusercontent.com/36038029/43526279-7be8fe7a-95ac-11e8-83c2-a89a90c9b70f.png)

**Comment by rowasc** [comment]
oh, can you try using https? I just remembered that http will always end up in that error due to a security update elsewhere in the system

**Comment by rowasc** [comment]
@Eve-wanderi how did this go? were you able to test it again?

**Comment by Eve-wanderi** [comment]
@rowasc no still got an error.

**Comment by rowasc** [comment]
unblocked. You can re-test

**Comment by rowasc** [comment]
@Eve-wanderi can you confirm if this worked for you? 
Thanks!

**Comment by Eve-wanderi** [comment]
@rowasc  still facing the issue.

![image](https://user-images.githubusercontent.com/36038029/43712530-2bbfefb8-997f-11e8-81bb-2d2bea42bd6e.png)

**Comment by rowasc** [comment]
<img width="1295" alt="screen shot 2018-08-06 at 10 43 21 am" src="https://user-images.githubusercontent.com/2434401/43719968-92a7f956-9965-11e8-9570-984b2a921a8d.png">
This is passing in QA.
